src/components/lib/Confirming.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+import Description from "./Description"
+import StatusIndicator from "../svgs/StatusIndicator"
+
+const Confirming = ({
+  className,
+  stepStatus,
+  error,
+  requiredConfirmations,
+  confirmations,
+  children,
+  extraMessage,
+}) => {
+  return (
+    <div className={className}>
+      <div className="page-top">
+        <StatusIndicator pulse />
+      </div>
+      <div className="page-body">
+        <div className="step">Step {stepStatus}</div>
+        <div className="title">
+          {error ? "Error confirming transaction" : "Confirming..."}
+        </div>
```

Comment:
Slightly nutty, but it occurs to me that we can probably abstract a `MintRedeemFlowPage` component with step + body, and a `MintRedeemFlowTitle` component with error vs true title handling, e.g.:

```
return (
  <MintRedeemFlowPage step="4">
    <MintRedeemFlowTitle error={error}>Confirming...</MintRedeemFlowTitle>

    <Description error={error}>
      <Confirming confirmations={confirmations} required={requiredConfirmations} />

      <p>We'll send you a browser notification...</p>
      <p><i>A watched block never boils</i></p>

      <div className="signer-fee">
        ...
      </div>
    </Description>
  </MintRedeemFlowPage>
)
```

Or something like that. `MintRedeemFlowPage` would probably also take some sort of attribute on what state the status indicator should be in.

Anyway, that doesn't have to happen now and may even be overkill. Just thinking out loud. Feels like it properly disentangles things that are unrelated to each other and gives a fairly straightforward content flow to the code.

Reply:
Ooo I like how you are thinking! Let's do it as a separate PR. Also, like you mentioned in the last comment, these views will change drastically soon but it will still probably make sense to create components like this.